packages/rum-angular/src/apm.service.ts

```diff
@@ -0,0 +1,108 @@
+/**
```

Comment:
`apm.service.ts` is not consistent with our naming convention it should be `apm-service`, even though it might be what is used in angular!